examples/basics/multiple_simulations.ipynb

```diff
@@ -0,0 +1,514 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "47607dd97aa922d1",
+   "metadata": {},
+   "source": [
+    " # Multiple Simulations on a Server"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2f7c912265a0ca4",
+   "metadata": {},
+   "source": [
+    "In this notebook we are going to perform multiple simulations sequentially using one NanoVer server.  <br>\n",
+    "Once they are loaded, the user can run the first one and then pass to the second, third ... etc., \n",
+    "via commands in the Jupyter notebook or the VR interface."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a3fb7e1bd99b669",
+   "metadata": {},
+   "source": [
+    "First, let's import the classes that we need from the NanoVer package:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "62aa87916b233f73",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-10-28T11:59:22.520269Z",
+     "start_time": "2024-10-28T11:59:20.687950Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from nanover.omni import OmniRunner\n",
+    "from nanover.omni.openmm import OpenMMSimulation\n",
+    "from nanover.omni.ase_omm import ASEOpenMMSimulation\n",
+    "\n",
+    "from examples.basics.visualisations_demos.visuals import clear_selections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d39b840d35707e4",
+   "metadata": {},
+   "source": [
+    "Then let's load the simulation files: we are loading three different protein ligand complexes:\n",
+    "1. Neuraminidase and zanamavir\n",
```

Comment:
```suggestion
    "1. Neuraminidase and zanamivir\n",
```
Check that the spelling is correct throughout the notebook

Reply:
Also, which input file are you using for this system? If you are not already doing so, it would probably be good to use the one adapted (and subsequently fixed) from the paper that the other input files have come from, rather than the version you made yourself (for consistency, and also in accordance with the information in the system_descriptions.md file)

Reply:
Actually the file is the Neuraminidase zanamivir file I have produced, but I didn't figure out how to initialize the protein inside the Nanover box 
( p.s.: it's just a visualization effect since implicit solvent simulation do not have boundary condition. )
But since we have the old file working I'll switch them.